core/src/main/scala/cell/Cell.scala

```diff
@@ -37,7 +41,7 @@ trait Cell[K <: Key[V], V] {
    * }}}
    *
    * @param other  Cell that `this` Cell depends on.
-   * @param valueCallback  Callback that receives the final value of `other` and returns an `Outcome` for `this` cell.
+   * @param valueCallback  Callback that receives the final value of `other` and returns an Outcome for `this` cell.
```

Comment:
When not referring to the type constructor, "Outcome" would have to be lower-case.